reflection/src/main/java/com/alevel_hw2/ContactDAO.java

```diff
@@ -0,0 +1,117 @@
+package com.alevel_hw2;
+
+import java.sql.*;
+
+public class ContactDAO {
+    private final String CREATE_TABLE_QUERY = "CREATE TABLE IF NOT EXISTS contacts (" +
+            "id bigserial PRIMARY KEY," +
+            "contact VARCHAR(255)," +
+            "contact_type VARCHAR(255))";
+
+    private final String CREATE_CONTACT = "INSERT INTO contacts (contact, contact_type) VALUES (?, ?)";
```

Comment:
Minor thing. Constants should be static as well.

```
private static final String CREATE_CONTACT;
```

or, you can move this constant directly to the method, to see this SQL query immediately and do not scroll up to the top of the DAO to see these constants.